### DISCUSSION TITLE
tab index

### DISCUSSION THREAD

**Discussion by CptOperations** [Budibase/budibase#3832] [Help]
Is there a way to alter the tab index of forms, mainly to set the buttons to follow fields in the index, to automate barcode scanning of products

**Comment by mjashanks** [comment]
There is no way to alter the tab-index. Tabbing follows the order in which the components are listed in the component tree on the left. 

I'd love to know more about this though - what do you mean by "...to set the buttons to follow fields in the index" ?

**Reply by CptOperations** [reply]
To clarify , my goal would be to have the 'save' button be next in the tab index after the last text input box, so when I program bar code reader I would tell it to tab once then submit enter key stroke , so it would save the record just from scanning the bar code

**Reply by mjashanks** [reply]
So, this works for me now, so long as I put my button at the bottom:

![image](https://user-images.githubusercontent.com/3524181/146804842-c61c4161-c6bf-4a52-bf8f-cf23b6986d01.png)

After I tab out of `Status`, pressing the space bar clicks the Save button.
